Game Studies.js

```diff
@@ -0,0 +1,119 @@
+{
+	"translatorID": "8c2c716c-09ba-421a-ae23-a1590d90a95e",
+	"label": "Game Studies",
+	"creator": "John Weis",
+	"target": "^https?://gamestudies\\.org/",
```

Comment:
might as well make this `"^https?://gamestudies\\.org/\\d"` since it covers all cases. The more strict this is the better.